src/forecast/prediction.py

```diff
@@ -0,0 +1,38 @@
+from sklearn.model_selection import train_test_split
+import pandas as pd
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import confusion_matrix, accuracy_score, classification_report
+from sklearn.naive_bayes import GaussianNB
+from sklearn.svm import SVC
+from sklearn.ensemble import GradientBoostingClassifier
+from sklearn.ensemble import RandomForestClassifier
+import pathlib as pl
+from sklearn.metrics import roc_auc_score, roc_curve
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+class Forecast:
```

Comment:
If it is going to be a class, probably better to be called Forecaster.

Reply:
changed accordingly